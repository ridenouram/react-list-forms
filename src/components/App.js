import React from 'react';
import ColorList from './ColorList';

export default function App() {
  return (
    <ColorList />
  );
}

