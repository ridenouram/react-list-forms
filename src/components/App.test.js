import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

describe('Color component', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
