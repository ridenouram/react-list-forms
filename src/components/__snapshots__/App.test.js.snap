// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component matches snapshot 1`] = `
<container
  className="Container"
>
  <Sidebar>
    <a
      href="#"
      name="dog"
      onClick={[Function]}
    >
      Dog
    </a>
    <a
      href="#"
      name="cat"
      onClick={[Function]}
    >
      Cat
    </a>
    <a
      href="#"
      name="snake"
      onClick={[Function]}
    >
      Snake
    </a>
    <a
      href="#"
      name="fish"
      onClick={[Function]}
    >
      Fish
    </a>
  </Sidebar>
  <p>
    dog
  </p>
</container>
`;
